config/daemon/systemd.md

```diff
@@ -131,12 +146,22 @@ you need to add this configuration in the Docker systemd service file.
     ```bash
     $ sudo systemctl daemon-reload
     ```
+     
+    Note: for [rootless](https://docs.docker.com/engine/security/rootless/) docker, 
+    ```bash
+    $ systemctl --user daemon-reload
+    ```
 
 5.  Restart Docker:
 
     ```bash
     $ sudo systemctl restart docker
```

Comment:
Should this also be `systemctl --user restart docker` ?

Reply:
that is the step 5.